### ISSUE TITLE
Bug - Cannot start apache2 service while file explorer is open.

### ISSUE THREAD

**Issue by emirarditi** [Microsoft/BashOnWindows#820]
I noticed a weird bug today. I have downloaded apache2 using the command sudo apt-get install apache2. After, I noticed that the service was not running. I tried sudo service apache2 start and got the following error: 
*[Mon Aug 08 18:45:10.538088 2016] [core:warn] [pid 2205] (22)Invalid argument: AH00076: Failed to enable APR_TCP_DEFER_ACCEPT

After that, I tried to close the file explorer window and ran the start command again and it worked!

Why the apache2 service does not start while the file explorer is open?

**Comment by aseering** [comment]
Hm, yeah, that is odd...  Just curious, is it repeatable?  Does it happen every time?

**Comment by emirarditi** [comment]
Yep I also tried in 2 different computers and got the same result.

**Comment by laklek** [comment]
Can confirm,
I had this issue too and thought it was because some network protocol not working in BashOnWindows, but doing closing explorer.exe seems to fix it.

**Comment by brnogz** [comment]
i also can confirm the issue

**Comment by erdenbatuhan** [comment]
Thank you, you've just saved my life! I had been going crazy about this for hours before I found your issue.

**Comment by therealkenc** [comment]
Ref #1953. `TCP_DEFER_ACCEPT` unimplemented surface whether `explorer.exe` is open or not; noting that `explorer.exe` is running whether it is on the screen or not, so I can't speak to that part.
